packages/next/package.json

```diff
@@ -64,7 +64,7 @@
     ]
   },
   "dependencies": {
-    "@babel/runtime": "7.15.3",
+    "@babel/runtime": "^7.15.3",
```

Comment:
Version locking is intentional to ensure that the version that was tested is being used. Feel free to update the dependency itself.

Reply:
@timneutkens I would like the assembly to have one version of the @babel/runtime package. In many packages, the version is not fixed and this leads to an increase in the build size.

Reply:
You can use yarn resolutions for that, but in practice dependencies should not have @babel/runtime anyway 🤔

Reply:
@timneutkens update fix version